zerver/lib/home.py

```diff
@@ -208,8 +208,14 @@ def build_page_params_for_home_page_load(
         # There is no event queue for spectators since
         # events support for spectators is not implemented yet.
         no_event_queue=user_profile is None,
+        server_sentry_dsn=settings.SENTRY_FRONTEND_DSN,
```

Comment:
Have you tested the spectators view for this? I ask mainly because we use the API to get some of `page_params` via it at the very bottom of `ui_init.js`, and I can imagine that somehow making this not work. (I think it should be OK, since we merge the two sources of `page_params`)

Reply:
Yup.  Here's an error from a spectator: https://zulip.sentry.io/issues/3974955637/?project=4504594953469952